frontend/catalyst/pennylane_extensions.py

```diff
@@ -1011,6 +1018,7 @@ class QJITDevice(qml.QubitDevice):
 
     name = "QJIT device"
     short_name = "qjit.device"
+    backend = ""
```

Comment:
I didn't really go for a new class just for abstracting the backend device name. Maybe it makes more sense when we're dealing with more device specifications from the frontend and more devices.

Reply:
@maliasadi does this have UI/UX/user-facing consequences? Or is this just an implementation detail?

Reply:
@josh146 Nothing is changed for the user except the ability of choosing between "lightning.kokkos" and "lightning.qubit" from the frontend. This's an implementation detail as far as I see.

Reply:
What I mean is declaring the member here in this way makes it a class attribute, meaning that all instances of this class share the same global value for `backend`. This is not a good idea because each instantiation of the class may want to use a different backend device.

Reply:
Ahh.. got your point! Fixed it. Thanks for catching this.